nix/haskell.nix

```diff
@@ -73,6 +73,9 @@ let
             # plc-agda is compiled from the Haskell source files generated from the Agda
             plc-agda.src = "${metatheory.plutus-metatheory-compiled}/share/agda";
 
+            language-plutus-core.components.benchmarks.language-plutus-core-create-cost-model.build-tools =
+              lib.mkForce [(pkgs.rWrapper.override { packages = with pkgs.rPackages; [tidyverse dplyr stringr MASS]; } )];
```

Comment:
Hm, I feel like we should be able to do this in a nicer way but looks fine for now.

Reply:
How would you do this nicer? I currently don't see another way.

Reply:
I mean specifically the `mkForce`. Might need some changes in `haskell.nix`, but I'd have thought those should merge...